source/Engine.h

```diff
@@ -188,6 +188,9 @@ class Engine {
 	// Viewport position and velocity.
 	Point center;
 	Point centerVelocity;
+	Point offset;
+	double offsetMultiplier = 0.;
+	constexpr static double offsetChange = 0.0025;
```

Comment:
This should probably go in Engine.cpp in the anonymous namespace with the other constants.

Reply:
will do this

Reply:
done